### PR TITLE
Fix duplicated word in subheading

### PR DESCRIPTION
**Pull request by savindi7**
### Related Issues
- https://github.com/wso2/product-is/issues/13537

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- None

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/server-configurations/pages/governance-connectors.tsx [hunk 1]
```diff
@@ -140,8 +141,12 @@ export const GovernanceConnectorsPage: FunctionComponent<GovernanceConnectorsPag
                     ? connectorCategory.description
                     : connectorCategory?.name
                     && t("console:manage.features.governanceConnectors.connectorSubHeading", {
-                        name: connectorCategory.name
-                    }))
+                        name: 
+                        categoryId == ServerConfigurationsConstants.OTHER_SETTINGS_CONNECTOR_CATEGORY_ID
```
Comment (brionmario) [inline apps/console/src/features/server-configurations/pages/governance-connectors.tsx:145]:
```suggestion
                        categoryId === ServerConfigurationsConstants.OTHER_SETTINGS_CONNECTOR_CATEGORY_ID 
```

File: apps/console/src/features/server-configurations/pages/governance-connectors.tsx [hunk 2]
```diff
@@ -81,9 +81,10 @@ export const GovernanceConnectorsPage: FunctionComponent<GovernanceConnectorsPag
         loadCategoryConnectors();
     }, []);
 
+    const path = history.location.pathname.split("/");
```
Comment (brionmario) [inline apps/console/src/features/server-configurations/pages/governance-connectors.tsx:84]: Add missing types to `path` & `categoryId`.

```suggestion
    const path = history.location.pathname.split("/");
```

File: apps/console/src/features/server-configurations/pages/governance-connectors.tsx [hunk 3]
```diff
@@ -140,8 +141,12 @@ export const GovernanceConnectorsPage: FunctionComponent<GovernanceConnectorsPag
                     ? connectorCategory.description
                     : connectorCategory?.name
                     && t("console:manage.features.governanceConnectors.connectorSubHeading", {
-                        name: connectorCategory.name
-                    }))
+                        name: 
+                        categoryId == ServerConfigurationsConstants.OTHER_SETTINGS_CONNECTOR_CATEGORY_ID 
+                            ? connectorCategory.name.split(" ")[0] :
```
Comment (brionmario) [inline apps/console/src/features/server-configurations/pages/governance-connectors.tsx:146]:
```suggestion
                            ? connectorCategory.name.split(" ")[0]
                            :  connectorCategory.name
```

File: apps/console/src/features/server-configurations/pages/governance-connectors.tsx (hunk 1 above)
Comment (savindi7) [reply apps/console/src/features/server-configurations/pages/governance-connectors.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/2987/commits/58a9084e0d55d1f34b7934f09b6fa06c328c22af

File: apps/console/src/features/server-configurations/pages/governance-connectors.tsx (hunk 3 above)
Comment (savindi7) [reply apps/console/src/features/server-configurations/pages/governance-connectors.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/2987/commits/58a9084e0d55d1f34b7934f09b6fa06c328c22af

File: apps/console/src/features/server-configurations/pages/governance-connectors.tsx (hunk 2 above)
Comment (savindi7) [reply apps/console/src/features/server-configurations/pages/governance-connectors.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/2987/commits/58a9084e0d55d1f34b7934f09b6fa06c328c22af

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/server-configurations/pages/governance-connectors.tsx [hunk 4]
```diff
@@ -81,9 +81,10 @@ export const GovernanceConnectorsPage: FunctionComponent<GovernanceConnectorsPag
         loadCategoryConnectors();
     }, []);
 
+    const path: string[] = history.location.pathname.split("/");
+    const categoryId: string = path[ path.length - 1 ];
```
[the remark this answers is not in the document]
Comment (savindi7) [reply apps/console/src/features/server-configurations/pages/governance-connectors.tsx:85]: Fixed with https://github.com/wso2/identity-apps/pull/3122/commits/66e141e87c7dbcfa1157a0a8268309cb43e75a42

**Pull request merged**